editions/tw5.com/tiddlers/community/resources/TiddlyWiki5 Squared by Iannis Zannos.tid

```diff
@@ -1,18 +1,18 @@
 created: 20141009170239174
-modified: 20141009170711343
+modified: 20190217000000000
 tags: Resources
 title: "TiddlyWiki5^2 documenting while learning TiddlyWiki5" by Iannis Zannos
 type: text/vnd.tiddlywiki
 url: http://larigot.avarts.ionio.gr/users/iani/wikis/tw5square.html
 
 A wealth of hints, tips and notes about using [[TiddlyWiki on Node.js]]:
 
-{{!!url}}
```

Comment:
Is there any reason not to keep the URL in the "url" field like we do for the other resource tiddlers?

Reply:
Fixed the URL pointing to the "url" field